src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx

```diff
@@ -22,65 +22,104 @@ freshnessValidatedDate: never
+<Callout variant="caution">
+  Delaying instantiation causes the agent to miss information until it is fully loaded and initialized. The order of execution of these scripts does lead to some loss in instrumentation data from the start of page views.
+</Callout>
 
-## What's next? [#whats-next]
+For possible performance or other reasons, you may wish to delay the initialization of the browser agent from the start of the page load cycle. You won't be able to use the `newrelic` API until the agent is fully initialized. This delay also creates incomplete data in your browser monitoring UI.
 
-Ideas for next steps: 
+Once the agent is fully initialized, it functions as expected. Any delay also affects when the agent wraps globals or subscribes to events relative to other libraries or code that affects captured data.
 
-* View your app data on the [Summary page](/docs/browser/browser-monitoring/getting-started/browser-summary-page).
-* Now that you've instrumented with the browser agent, go deeper with your data with our [Improve your website tutorial](/docs/journey-performance/guide-to-monitoring-core-web-vitals). 
-* Learn how to optimize your browser monitoring experience with our [Best practices guide for browser monitoring](/docs/new-relic-solutions/best-practices-guides/full-stack-observability/browser-monitoring-best-practices-guide). 
+You can delay the agent initialization in these ways:
 
-## Install methods [#choose-deployment]
+* Add a `script-src` tag with the `async` attribute.
+* Add a `script-src` tag with the `defer` attribute.
+* Use dynamic import or chronologically late placement of the loader code.
 
-When it comes to adding the browser agent to your webpages, you have two deployment methods: 
+Your page's `src` must point to a valid agent loader version in our CDN. For example, the URL that always contains our current Pro+SPA agent release is `https://js-agent.newrelic.com/nr-loader-spa-current.min.js`. [CSP is required](/docs/browser/new-relic-browser/getting-started/compatibility-requirements-browser-monitoring#csp) to load external third-party scripts.
 
-* APM + browser monitoring: Use one of our APM agents to auto-instrument browser monitoring 
-* Browser monitoring only: Add our browser agent manually as a JavaScript snippet to your front-end HTML 
+To determine when you can start calling the `newrelic` API, subscribe to an event at the global scope:
 
-Here's some guidance on these two methods. 
+```js
+window.addEventListener('newrelic', (evt) => {
+  if (evt.detail.loaded) {
+    // you can start using newrelic.interaction(), etc. now
+  }
+})
+```
+    
+</Collapser>
+        </TabsPageItem>
+        <TabsPageItem id="2">
+## Inject browser agent via APM [#install-apm]
 
-### APM + browser monitoring [#apm-agent]
+With an APM agent, you can deploy browser monitoring without manually adding the snippet to your code.
```

Comment:
I would remove this sentence since it's basically repeated in the following sentences

Reply:
I ended up rewriting these two paragraphs to address this